app/models/visitor.rb

```diff
@@ -188,4 +190,28 @@ def visit_command(node)
 
     super
   end
+
+  private
+
+  # handle method names like `Skiptrace.current_bindings` where the constant
+  # wasn't already visited
+  # rubocop:disable Lint/AssignmentInCondition
+  def inferred_context_for(target)
+    if existing = @analyzer.modules.detect { |name| name.qualified_name == target }
+      return prev if @modules.find { |mod| mod.qualified_name == existing.qualified_name }
+
+      @modules << existing
+      @namespace << existing
```

Comment:
I think we shouldn't add add it to the `@modules` or `@namesapce` since we are not going to traverse it's children, so we don't need to set setup the namespace